### PR TITLE
Completed string-manipulation-2 assignment

### PR DESCRIPTION
**Pull request by Aiden-Peace300**
image

### CODE REVIEW

File: string-manipulation-2/src/first-chars.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+    for (let i = 0; i < string.length; i++) {
+      result += string[i];
+    }
+  } else if (string.length >= length) {
+    for (let i = 0; i < length; i++) {
+      result += string[i];
+    }
+  }
+  return result;
+}
```
Comment (shawnkost) [inline string-manipulation-2/src/first-chars.js:18]: We can simplify this using the `slice` method
```
function firstChars(length, string) {
  return string.slice(0, length);
}
```

File: string-manipulation-2/src/last-chars.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+/* exported lastChars */
+function lastChars(length, string) {
+  let result = '';
+
+  if (string.length < length || string.length >= length) {
+    result = string.slice(-length);
+  }
+
+  return result;
+}
```
Comment (shawnkost) [inline string-manipulation-2/src/last-chars.js:10]: Alternative solution!
```
function lastChars(length, string) {
  const startIndex = Math.max(0, string.length - length);
  return string.slice(startIndex);
}
```

**Review by shawnkost — APPROVED** [review]
👍🏻

File: string-manipulation-2/src/first-chars.js (hunk 1 above)
Comment (Aiden-Peace300) [reply string-manipulation-2/src/first-chars.js]: Updated! Thank you!

**Pull request merged**